client/types.go

```diff
@@ -21,7 +21,7 @@ type infoPeersResponse struct {
 
 type ContractInfo struct {
 	Symbol   string `json:"symbol"`
-	Decimals uint8  `json:"decimals"`
+	Decimals int32  `json:"decimals"`
```

Comment:
Why not just do `uint` or `int` here? by default it is already 32.

Reply:
removed it altogether and am just using `types.Currency`: 0d901a2